<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_previewClass</class>
 <widget class="QWidget" name="page_previewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>page_preview</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#page_previewClass
{
    border-image: url(:/page_preview/Resources/BackGround2.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,500,0,300">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_2
{
    border-image: url(:/page_preview/Resources/BackGround2.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="Top">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>书名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>作者</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>出版社</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE3"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>分类</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">background: #EDF8FF;
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size: 12px;
font-family: Microsoft YaHei;</string>
              </property>
              <property name="text">
               <string>仅显示有余书籍</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Down" stretch="2,2,1,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>价格</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SP1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
            </property>
            <property name="maximum">
             <number>4396</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SP2">
            <property name="styleSheet">
             <string notr="true">
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
            </property>
            <property name="minimum">
             <number>-4</number>
            </property>
            <property name="maximum">
             <number>4396</number>
            </property>
            <property name="value">
             <number>4396</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1,0,1">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>出版日</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SP3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>2025</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SP4">
            <property name="styleSheet">
             <string notr="true">
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
            </property>
            <property name="minimum">
             <number>1800</number>
            </property>
            <property name="maximum">
             <number>2025</number>
            </property>
            <property name="value">
             <number>2025</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="PB_search">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #EDF8FF;
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_clean">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #EDF8FF;
border: 1px solid #CFDDEF;
border-top-left-radius:6px;
border-bottom-left-radius:6px;

font-size: 14px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #5786C7;</string>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="Label_Src_Msg">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="600">
     <item>
      <widget class="QTableWidget" name="TableW_book">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/page_preview/Resources/BackGround4.png);</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="L_current_honndana">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>当前：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PB_syurui_manager">
       <property name="styleSheet">
        <string notr="true">width: 120px;
height: 28px;
background: #06d2ff;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #ffffff;
line-height: 60px;

</string>
       </property>
       <property name="text">
        <string>分类管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_book_manager">
       <property name="styleSheet">
        <string notr="true">width: 120px;
height: 28px;
background: #7470BC;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
       </property>
       <property name="text">
        <string>书籍管理</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="100">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">#widget
{
    border-image: url(:/page_preview/Resources/BackGround2.png);
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 14px;
	font-family: Microsoft YaHei;</string>
              </property>
              <property name="text">
               <string>书籍总数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LB1">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 18px;
	font-family: Microsoft YaHei;
color: rgb(85, 170, 0);</string>
              </property>
              <property name="text">
               <string>10种</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 14px;
	font-family: Microsoft YaHei;</string>
              </property>
              <property name="text">
               <string>分类总数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LB2">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 18px;
	font-family: Microsoft YaHei;
color: rgb(85, 170, 0);</string>
              </property>
              <property name="text">
               <string>4396种</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 14px;
	font-family: Microsoft YaHei;</string>
              </property>
              <property name="text">
               <string>库存总数:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LB3">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 18px;
	font-family: Microsoft YaHei;
color: rgb(85, 170, 0);</string>
              </property>
              <property name="text">
               <string>1000本</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 14px;
	font-family: Microsoft YaHei;</string>
              </property>
              <property name="text">
               <string>租赁总数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LB4">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">	font-size: 18px;
	font-family: Microsoft YaHei;
color: rgb(85, 170, 0);</string>
              </property>
              <property name="text">
               <string>10本</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="TableW_book2">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="page_preview.qrc"/>
 </resources>
 <connections/>
</ui>
